### DISCUSSION TITLE
IndieAuth discussion

### DISCUSSION THREAD

**Discussion by mattl** [foocorp/hacienda#97] [Future enhancements]
See #36 

We'd do something similar for IndieAuth. IndieAuth lets you enter a personal website URL like https://mat.tl as your login. Links to other platforms in your website's code will identify you with other services. 

For example, I have `<link rel="me" href="https://github.com/mattl" />` on my website, which redirects me to GitHub for logins. I could also be redirected to my Libre.fm profile if support was added. 

@foocorp/beta-testers Do you have a personal website? Does it support IndieAuth?

Test your own site here: https://indiewebify.me/validate-rel-me/?url=https%3A%2F%2Fmat.tl%2F

**Comment by fluffy-critter** [comment]
> For example, I have `<link rel="me" href="https://github.com/mattl" />` on my website, which redirects me to GitHub for logins

Note that this describes RelMeAuth, which is related to but distinct from IndieAuth, and is much more difficult for client operators (i.e. login consumers) to support. True IndieAuth requires a bit more setup, and unfortunately the documentation around this is super confusing to newcomers.
